repository/g.json

```diff
@@ -1627,6 +1627,19 @@
 			]
 		},
 		{
+  		"name": "Green Writing Color Scheme",
+        	"details": "https://github.com/caoyeacy/sublime-writing-color-scheme-green",
+                "description": "Green Writing Color Scheme For Writing LaTeX",
+                "labels": ["color"],
+  		"releases": [
+  				{
+  				"sublime_text": "*",
+  	                	"branch": "master",
+                                "tags": true
```

Comment:
You can use both tag and branch-based releases. In fact, we no longer accept new packages using branch-based releases.